frontend/src/pages/sites/sites.tsx

```diff
@@ -131,19 +122,28 @@ const Sites: React.FC = () => {
               </div>
               <h2>Landing</h2>
```

Comment:
Need to add the title style margin-top: 0

Reply:
is this resolved?